### PR TITLE
Pootle data checks

### PR DESCRIPTION
**Pull request by phlax**
Adds checks data tables for store and tp, to aggregate check counts

### CODE REVIEW

File: pootle/apps/pootle_data/abstracts.py [hunk 1]
```diff
@@ -77,3 +79,21 @@ class Meta(object):
...
+        max_length=64,
+        db_index=True)
+    category = models.IntegerField(
+        null=False,
+        default=Category.NO_CATEGORY)
+    count = models.IntegerField(
+        null=False,
+        blank=False,
+        default=0,
+        db_index=True)
```
Comment (unho) [inline pootle/apps/pootle_data/abstracts.py]: Why index the count?

Comment (phlax) [inline pootle/apps/pootle_data/abstracts.py]: not sure - maybe no point

Comment (phlax) [inline pootle/apps/pootle_data/abstracts.py]: @unho seems there is point http://sqlmag.com/database-performance-tuning/indexing-dos-and-don-ts

**Comment by unho** [comment]
lgtm

**Pull request merged**